<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureAnalyzerDialogBase</class>
 <widget class="QDialog" name="TextureAnalyzerDialogBase">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Texture Analyzer 1.0</string>
  </property>
  <widget class="QComboBox" name="box_banda_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>303</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>273</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Imagens</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>283</y>
     <width>470</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_banda_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>303</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banda  1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_banda_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>333</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_banda_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>333</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banda  2</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_banda_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>363</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_banda_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>363</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banda  3</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_banda_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>393</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_banda_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>393</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banda  4</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_banda_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>423</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_banda_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>423</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banda  5</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_banda_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>453</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_banda_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>453</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banda  6</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_banda_7">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>483</y>
     <width>470</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_banda_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>483</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Banda  7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Quadro de Mensagens</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>14</y>
     <width>380</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="logboard">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>530</width>
     <height>210</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_export">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>250</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_compute">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>600</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Calcular Texturas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_quit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>600</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Parâmetros</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>450</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_mask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>533</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Máscara</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="box_mask">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>533</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>563</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Outputs</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="path_output">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>564</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_output">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>561</y>
     <width>35</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_classify">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>600</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Classificar Texturas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_clear">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Log</string>
   </property>
  </widget>
  <widget class="QLabel" name="log_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>140</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>200</width>
     <height>13</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>3</width>
     <height>610</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>14</y>
     <width>460</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Texturas</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>35</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Textura</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_text">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>35</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>90</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Número de pixels: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Média:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>110</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Variância:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_pix">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_med">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>110</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_var">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>719</x>
     <y>160</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_title_5">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>158</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Histograma das classes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_n_classes">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>533</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Número de Classes</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="box_num_classes">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>533</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="class_name">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_add_class">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>33</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Adicionar ao QGIS</string>
   </property>
  </widget>
  <widget class="QWidget" name="mplwindow" native="true">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>180</y>
     <width>530</width>
     <height>381</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="mplvl"/>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>530</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>928</x>
     <y>571</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Desenvolvido por Givaldo Cesar</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>898</x>
     <y>586</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Engenheiro Cartógrafo e Agrimensor&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>948</x>
     <y>601</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Email: givaldocesar@live.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="lasenso">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>580</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="decart">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>580</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ctg">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>575</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ufpe">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>575</y>
     <width>45</width>
     <height>50</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
